aidial_analytics_realtime/rates.py

```diff
@@ -65,13 +65,22 @@ def __init__(self, rates_str: str | None = None):
 
     def calculate_price(
         self,
+        deployment: str,
         model: str,
         request_content: str,
         response_content: str,
         usage: dict | None,
     ) -> Decimal:
+        deployment_rate = self.rates.get(deployment)
+        if deployment_rate:
+            return deployment_rate.calculate(
+                request_content, response_content, usage
+            )
+
         model_rate = self.rates.get(model)
-        if not model_rate:
-            return Decimal(0)
+        if model_rate:
+            return model_rate.calculate(
```

Comment:
I would prefer to avoid copy-pasting `rate.calculate` call.
This logic has two steps: choosing the correct rate and calculating the price using it. I would prefer not mixing it.

Reply:
fixed